content/blog/2022-08-16-august-22-heartbeat.md

```diff
@@ -0,0 +1,360 @@
+You can find more about his workflow in
+[his repo for the project.](https://github.com/mikolajkania/kaggle-03-house-prices)
+
+![Using DVC for Kaggle Competition](/uploads/images/2022-08-16/kaggle-dvc.png '=800')
+_DVC with Kaggle
+([Source link](https://mikolajkania.com/2022/08/07/dvc-kaggle-mlops/))_
+
+Mikołaj explains how DVC's project structure ensures reproducible results and
+develops habits on best practices. One drawback he noted was the lack of an
+experimentation UI, but we just introduced the DVC extension for VS Code to help
+with that, and there’s always Iterative Studio. Look out for improvement to the
+experiment features in both tools in the coming months! Also, experimenting with
+DVC in Kaggle may give you some good practice for things we are cooking up
+internally! 😉🤫
+
+## **Shambhavi Mishra** - Searching for Semantic Similarity
+
+[**Shambhavi Mishra**](https://twitter.com/ShambhaviCodes) in her post
+[Searching for Semantic Similarity](https://medium.com/towards-artificial-intelligence/searching-for-semantic-similarity-cfbff2388d04)
+details the steps of her NLP project on similarity algorithms. She mainly
+focuses on cosine similarity using a Stackoverflow questions dataset. The
+end-to-end project uses Sentence BERT, Fast Text, DVC, DAGsHub, Streamlit and
+deploys the web app on an AWS EC2 instance.
+
+Once you follow all the steps you will have computed the similarity between a
+search query and a database of texts and rank all the data by their similarity
+score to retrieve the most similar text to its index.
+
+![Cosine Similarity](/uploads/images/2022-08-16/cosine-similarity.png '=800')
+_Understanding Cosine Similarity
+([Source link](https://www.oreilly.com/library/view/mastering-machine-learning/9781785283451/ba8bef27-953e-42a4-8180-cea152af8118.xhtml))_
+
+## **Evgenii Munin** - Run S3 Locally With MinIO for the DVC Machine Learning Pipeline
+
+If you are in need of object storage to work with data through an API, but need
+to do so in a private network,
+[**Evgenii Munin**](https://www.linkedin.com/in/evgenii-munin-01932a143/) shows
+how to set up MinIO as remote storage with DVC to do just that
+[in this piece in Medium](https://betterprogramming.pub/run-s3-locally-with-minio-for-dvc-machine-learning-pipeline-7fa3d240d3ab).
+In this cool use case, he starts with installing the MinIO server and builds a
+Docker image to run it, sharing a great repo on Kafka-to S3 where MinIO was used
+to mock the S3 for the data. Then he shows you how to link the MinIO server as
+DVC remote storage.
+
+![Minio Browser](/uploads/images/2022-08-16/minio.png '=800') _Minio Browser
+with Data pushed from DVC
+([Source link](https://betterprogramming.pub/run-s3-locally-with-minio-for-dvc-machine-learning-pipeline-7fa3d240d3ab))_
+
+## **Caleb Kaiser** - Moving from Data Science to Machine Learning Engineering
+
+It can sometimes be confusing to determine where data science stops and machine
```

Comment:
Could you break this up into a couple of paragraphs? It'll break up the wall of text.